JMS/README.md

```diff
@@ -109,6 +109,29 @@ Which will download and copy the dependencies into the current directory.
 
 
 ### Building the samples with maven
+If you are using a Windows machine to build these samples , a symbolic link between the `./src/main/java/com` and `./com` directory is necessary.
+Navigate to `src/main/java` and run the following command to remove existing com file.
+````
+del com
+````
+To create a symbolic link between `./src/main/java/com` and `./com` use the following command.
+````
+mklink /J com ..\..\..\com
+````
+If you use maven to build the samples on a Windows machine , the classpath separator needs to be `;` and not `:`.
+eg.
+````
+java -cp target/mq-dev-patterns-0.1.0.jar: com.ibm.mq.samples.jms.JmsPut
```

Comment:
This is an example where there is only 1 entry in the classpath, and the separator is redundant, ie it can be removed, and hence will work with both linux and windows. 

You have correctly edited all such occurrences in this readme, to remove the redundant separator. 

Instead provide an example where the separator is still being used and is necessary. 

eg. 
`javac -cp ./com.ibm.mq.allclient-9.2.5.0.jar:./javax.jms-api-2.0.1.jar:./json-20230227.jar:. com/ibm/mq/samples/jms/JmsPut.java`

Although for that example we should be using the latest current version of the allclient (9.2.4)